clientv3/kv.go

```diff
@@ -87,30 +88,44 @@ type Txn interface {
 }
 
 type kv struct {
+	conn   *grpc.ClientConn // conn in-use
```

Comment:
but activeConn might not be the one KV is using. so we might end up switch a good conn when there are multiple concurrent failures.